.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: csharp
+
+os:
+  - osx
+
+install:
+    - ps: |
+        Install-Module -Name Pester -Force
+
+test_script:
+    - ps: |
+        Import-Module Pester
+        Invoke-Pester
```

Comment:
This will take a bit more work. My guess is Powershell needs to be installed through apt-get addons for Linux and Homebrew on OSX (we can just do OSX for now because I didn't add support for Linux).

Reply:
Yeah I kind of gave up on Travis at the moment 😛

Reply:
If you enable it I can contribute a config

Reply:
Enabled!

Reply:
I guess I can just use the `.travis.yml` from the Pester PR you referenced.

Reply:
That one is not good ;) I'm writing one